Logical operators exercise

danpottshimself opened the pull request:
Completed all tasks.

dpotts7 committed:
Added Comparsion+and+Logical+Operators exercise.

Completed task 1/4.

Declaring two variables and outputting the largest.

dpotts7 committed:
Logical Operators completed.

3/3 Tasks shown.

keithbarrow commented on logical-operators.js:
```diff
@@ -0,0 +1,44 @@
+
+//Declare three variables and output the product and whether product is negative or not.
+var negcheck = function(value1,value2,value3) {
+    var answer = (value1 * value2) * value3;
```
No need for the brackets - order is unimportant!:
a \* b \* c = (a_b) \* c = a_ (b *c)

keithbarrow commented on logical-operators.js:
```diff
@@ -0,0 +1,44 @@
+    console.log(smallNumber)
+}
+
+//Declare three variables and output the product and whether product is negative or not.
```
No comments.

keithbarrow commented on logical-operators.js:
```diff
@@ -0,0 +1,44 @@
+/**
```
No Comments

keithbarrow commented on logical-operators.js:
```diff
@@ -0,0 +1,44 @@
+        console.log("Product is negative!");}}
+negcheck(8,-3,2);
+
+//A grading system: if the score provided is less that 40 = F, less than 50= E etc up to A. No A*, they’re just a whack idea to cover up the fact exams are too easy.
```
No Comments

keithbarrow commented on logical-operators.js:
```diff
@@ -0,0 +1,44 @@
+    if (check < 40) {
+        console.log("You got an F");
+    }
+        else if(check < 50) {
```
please sort this alignment out, not right

keithbarrow commented on logical-operators.js:
```diff
@@ -0,0 +1,44 @@
+               else if (check <70){
+                    console.log("You got a B")}
+    else {
+        console.log("You are a smarty pants.");}}
```
Not what was asked for...

dpotts7 committed:
Made the requested changes.
Fixed:
-Structure of the code in certain areas.
-Removed unnecessary brackets.
-Removed features that were not required.

keithbarrow commented on logical-operators.js:
```diff
@@ -0,0 +1,39 @@
+var negcheck = function(value1,value2,value3) {
+var answer = value1 * value2 * value3;
+console.log("The product is" + answer);
+    if (answer > 0) {
```
What if the answer is 0?

dpotts7 committed:
I have changed the if statement to show that 0 is in fact a positive number.
(I have learnt something new today).

keithbarrow commented:
going to ignore the comments - just want to get this stuff out the way :shipit:
